hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -139,8 +142,8 @@ spec = around showLogsOnFailure $ do
                 seedFromFaucet_ node carolCardanoVk 100_000_000 Fuel (contramap FromFaucet tracer)
 
                 send n1 $ input "Init" []
-                waitFor tracer 10 [n1, n2, n3] $
-                  output "ReadyToCommit" ["parties" .= Set.fromList [alice, bob, carol]]
+                waitForAllMatch 10 [n1, n2, n3] $
+                  headIsInitializedWith (Set.fromList [alice, bob, carol])
```

Comment:
Should: Rename also these identifiers to match the actual server output name, i.e. `headIsInitializingWith`